### PR TITLE
reCaptcha does not validate on server side

### PR DESCRIPTION
**Pull request by valdisiljuconoks**
reCaptcha response received back after client side validation is present in "g-recaptcha-response" field.

https://www.google.com/recaptcha/admin#site/337671016?setup

### CODE REVIEW

File: ClientResources/ViewMode/RecaptchaElementBlock.js [hunk 1]
```diff
@@ -46,11 +46,13 @@ function initRecaptchaElements() {
         $(".Form__Element.FormRecaptcha").each(function (index, element) {
             var $element = $(element),
                 $widgetContainer = $(".g-recaptcha", $element),
-                siteKey = $element.data("epiforms-sitekey");
+                siteKey = $element.data("epiforms-sitekey"),
+                initialized = $element.data("epiforms-initialized");
```
Comment (lockevn) [inline ClientResources/ViewMode/RecaptchaElementBlock.js]: you check for "epiforms-initialized"
but you set to "epiforms-recaptcha-initialized"

is it mistake?

Comment (valdisiljuconoks) [inline ClientResources/ViewMode/RecaptchaElementBlock.js]: @lockevn yes. good catch. I fixed in my local version.

**Comment by valdisiljuconoks** [comment]
Fixed. Conflict in .sln file

**Comment by Gaubeou** [comment]
Hi Valdis Iljuconoks, I'm from the AddOns team. 
Thank you for your support, I created a bug fix based on your pull request. We have to pass QA (tester) processes before we can apply it :)

**Comment by Gaubeou** [comment]
Valdis Iljuconoks, could you give us more information to reproduce this bug? As I tested, the validation on server side still works fine.

**Comment by valdisiljuconoks** [comment]
@Gaubeou this was while ago. AFAIR we got couple of issues:
* post back of the reCaptcha value could not be read from `submitData[this.Content.GetElementName()]`. this was fixed in https://github.com/episerver/EPiServer.Forms.Samples/pull/6/commits/1b2ce1cb6d9f6c1b123870fee95f2e9e05750589.
* sometimes reCaptcha control on client side was initialized multiple times (could not understand reasons behind this - some reporting here as well: https://github.com/episerver/EPiServer.Forms.Samples/issues/2). so this was fixed in https://github.com/episerver/EPiServer.Forms.Samples/pull/6/commits/442aa47251422ff8754d4c93bd0b81e7127ec0ef

Rest of PR is just formatting, .sln file and other stuff.

**Comment by valdisiljuconoks** [comment]
Could it be that this is fixed in latest version?

**Pull request closed without merging**